install_config/install/quick_install.adoc

```diff
@@ -18,77 +18,215 @@ ifdef::openshift-enterprise[]
+  public_ip: 24.222.0.1 <6>
+  public_hostname: master.example.com <6>
+  master: true <7>
+  node: true <7>
+  containerized: true <8>
+- ip: 10.0.0.2
+  hostname: node1-private.example.com
+  public_ip: 24.222.0.2
+  public_hostname: node1.example.com
+  node: true
+- ip: 10.0.0.3
+  hostname: node2-private.example.com
+  public_ip: 24.222.0.3
+  public_hostname: node2.example.com
+  node: true
 ----
+<1> The OpenShift variant to install. For OpenShift Enterprise, set this to
+`openshift-enterprise`.
+<2> A valid version your selected variant. If not specified, this defaults to
+the newest version for the specified variant. For example: `3.1` or `3.0`.
+<3> Defines which user Ansible uses to SSH in to remote systems for gathering
+facts and for the installation. By default, this is the root user, but you can
+set it to any user that has *sudo* privileges.
+<4> Defines a list of the hosts onto which you want to install the OpenShift
+master and node components.
+<5> Required. Allows the installer to connect to the system and gather facts
+before proceeding with the install.
+<6> Required for unattended installations. If these details are not
+specified, then this information is pulled from the facts gathered by the
+installation utility, and you are asked to confirm the details. If undefined for
+an unattended installation, the installation fails.
+<7> Determines the type of services that are installed. At least one of these
+must be set to *true* for the configuration file to be considered valid.
+<8> If set to *true*, OpenShift services are run in a container on this host
+instead of installed using RPM packages.
+====
+
+[[running-an-unattended-installation]]
+
+== Running an Unattended Installation
+
+[NOTE]
+====
+Ensure you have read through the link:#quick-install-prereqs[Prerequisites]
+before proceeding.
+====
+
+Unattended installations allow you to define your hosts and cluster
+configuration in an
+link:#defining-an-installation-configuration-file[installation configuration
+file] before running the installation utility so that you do not have to go
+through all of the link:#running-an-interactive-installation[interactive
+installation] questions and answers. It also allows you to resume an interactive
+installation you may have left unfinished, and quickly get back to where you
+left off.
+
+To run an unattended installation, first define an
+link:#defining-an-installation-configuration-file[installation configuration
+file] at *_~/.config/openshift/installer.cfg.yml_*. Then, run the installation
+utility with the `-u` flag:
 
-. Follow the on-screen instructions to install a new OpenShift Enterprise
-instance.
+----
+$ atomic-openshift-installer install -u
+----
 
-*Method 2: Downloading and Running the Installation Utility*
+The installation utility uses the configuration specified in the
+*_~/.config/openshift/installer.cfg.yml_* file by default to run the installation,
+and fails if the file does not exist.
```

Comment:
By default, the installation utility uses the configuration specified in the *_~/.config/openshift/installer.cfg.yml_* file. If this default file location is used, then it is automatically read and reused every time you run the installation. You can specify a different location for the file with the `-c` option, but doing this will require you to specify the same `-c` file location every time you run the installation.